src/ontology/mondo-edit.obo

```diff
@@ -544744,6 +544745,28 @@ xref: NCIT:C34629
 is_a: MONDO:0021201 {source="NCIT:C34629"} ! skin infection
 property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/8306" xsd:anyURI
 
+[Term]
+id: MONDO:0100599
+name: POLR-related leukodystrophy
+def: "A rare hypomyelinating leukodystrophy disorder characterized by the association of dental abnormalities (delayed dentition, abnormal order of dentition, hypodontia), hypogonadotropic hypogonadism, and hypomyelinating leukodystrophy manifesting with neurodevelopmental delay or regression and/or progressive cerebellar symptoms." [Orphanet:289494]
+subset: ordo_disorder {source="Orphanet:289494"}
+subset: orphanet_rare {source="Orphanet:289494"}
+synonym: "4H leukodystrophy" EXACT [Orphanet:289494]
+xref: Orphanet:289494 {source="MONDO:equivalentTo"}
+xref: UMLS:C5679947 {source="MEDGEN:1803536", source="MONDO:equivalentTo", source="MONDO:MEDGEN"}
+is_a: MONDO:0019046 {source="Orphanet:289494"} ! leukodystrophy
+property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/8351" xsd:anyURI
```

Comment:
Based on the name, it seems like the disease should be defined based on the gene affected. For example "~in which the cause of the disease is a variation in any of the POLR genes, including POLR3, POLR1,... 
Note that I don't know which one it would cover (need more investigation). 

Also Note that we cannot have an axiom such as "gene1 OR gene2 OR gene3"... (https://github.com/monarch-initiative/mondo/pull/8173)